ibis/pandas/aggcontext.py

```diff
@@ -421,7 +421,7 @@ def __init__(self, preceding, *args, **kwargs):
             )
             else 'both'
         )
-        super().__init__('rolling', preceding, *args, closed=closed, **kwargs)
+        super().__init__('rolling', preceding, *args, closed=closed, min_periods=1, **kwargs)
```

Comment:
Should we add `min_periods=1` to `Cumulative` as well?

Reply:
We could do this in a follow-up as well.

Reply:
Just curious why does Cumulative needs min_period?

Reply:
I see, it's one of the parameter sort of unique to expanding window in pandas:
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.expanding.html

Reply:
I will create a follow-up to add `min periods=1` to `Cumulative`

Reply:
Actually do we need to even add `min_periods=1` to Cumulative? It looks like that is already the default behavior for Expanding windows.

Reply:
Cool, then we don't need to do that. Thanks for looking into it.